### PR TITLE
[#3242]-profile-validation-added (connects #3242)

### PR DESCRIPTION
**Pull request by yuviii**

### CODE REVIEW

File: src/app/users/users-achievements/users-achievements-update.component.ts [hunk 1]
```diff
@@ -161,14 +161,23 @@ export class UsersAchievementsUpdateComponent implements OnInit {
 
   onSubmit() {
     this.editForm.updateValueAndValidity();
-    if (this.editForm.valid) {
-      this.updateAchievements(this.docInfo, this.editForm.value, { ...this.user, ...this.profileForm.value });
-    } else {
+    this.profileForm.updateValueAndValidity();
+    console.log(this.profileForm.valid);
```
Comment (lmmrssa) [inline src/app/users/users-achievements/users-achievements-update.component.ts]: please remove console.log

File: src/app/users/users-achievements/users-achievements-update.component.ts [hunk 2]
```diff
@@ -161,14 +161,23 @@ export class UsersAchievementsUpdateComponent implements OnInit {
...
+    if (!this.editForm.valid) {
       Object.keys(this.editForm.controls).forEach(field => {
         const control = this.editForm.get(field);
         control.markAsTouched({ onlySelf: true });
       });
     }
+    if (!this.profileForm.valid) {
```
Comment (lmmrssa) [inline src/app/users/users-achievements/users-achievements-update.component.ts]: these can go under single else condition as before

**Comment by lmmrssa** [comment]
@yuviii On the image birthdate is validating but not showing any message. So could you also fix this
screen shot 2019-02-25 at 15 35 58

**Comment by yuviii** [comment]
@lmmrssa please review the update

**Pull request merged**